docs/source/executing-operations/subscription-callback-protocol.mdx

```diff
@@ -273,54 +275,4 @@ The following are common error states that can occur with this protcol:
 - **Emitter** receives an error HTTP status code from **Router**'s callback endpoint. 
   - If the error code is 404, **Emitter** should consider the associated subscription terminated by **Router**.
   - In other error cases, **Emitter** should consider the subscription terminated due to an unexpected error.
-- **Emitter** fails to send **Router** a [`check`](#check) message for an active subscription every five seconds, causing **Router** to terminate that subscription.
-
-## Examples
```

Comment:
i think it was nice to have a step by step example of the protocol execution

Reply:
Actually there is already an example in the same document. Could you read it and let me know if you think it's enough ?

Reply:
👍 didn't notice the other section, yeah this is redundant